known.json

```diff
@@ -16,7 +16,8 @@
 	"3040783849904107057L": { "owner": "Coincheck.com", "description": "Main Wallet" },
```

Comment:
Please make a distinction between both wallets, i.e. don't describe them both as "Main Wallet". Maybe "Old Wallet" and "Main Wallet".